tests/unittests/engine/operator_runner/test_concat_runner.py

```diff
@@ -74,8 +71,7 @@ def test_concat_runner(self):
             writer
         )
         runner.set_op(ConcatOperator('row'))
-        t = threading.Thread(target=run, args=(runner, ))
-        t.start()
+        t = start_runner(runner)
```

Comment:
I think we need a stable checker for the runner.

Reply:
Only for UT, because the ut will check the runner status after the runner start.